c2rust-analyze/src/context.rs

```diff
@@ -357,7 +361,16 @@ impl<'a, 'tcx> AnalysisCtxt<'a, 'tcx> {
     }
 
     pub fn project(&self, lty: LTy<'tcx>, proj: &PlaceElem<'tcx>) -> LTy<'tcx> {
-        util::lty_project(lty, proj)
+        let adt_func = |adtdef: AdtDef, field: Field| {
+            let fielddef = adtdef.all_fields().nth(usize::from(field)).unwrap();
+            let fielddef_name = fielddef.name;
+            eprintln!("projecting into {adtdef:?}.{fielddef_name:}");
```

Comment:
Is there any logging we should be using or is `eprintln!` fine for now?

Reply:
I've just been using `eprintln!` for now, with the assumption that we'll someday set up a proper logging framework and do a big search-and-replace to convert them all to `debug!`/`trace!`.